src/components/calcite-block/calcite-block.tsx

```diff
@@ -145,13 +145,20 @@ export class CalciteBlock {
     const toggleLabel = open ? intlCollapse || TEXT.collapse : intlExpand || TEXT.expand;
 
     const hasIcon = getSlotted(el, SLOTS.icon);
+    const collapseIcon = open ? TOGGLE_ICON.opened : TOGGLE_ICON.closed;
+
     const headerContent = (
       <header class={CSS.header}>
         {hasIcon ? (
           <div class={CSS.icon}>
             <slot name={SLOTS.icon} />
           </div>
         ) : null}
+        {collapsible ? (
+          <div class={CSS.toggleIcon} aria-hidden="true">
```

Comment:
nitpick: Is this div needed or could the class and aria-hidden be applied directly to the calcite-icon?

Reply:
yeah...prolly.